impl/src/main/java/org/ehcache/internal/executor/DefaultThreadPoolsService.java

```diff
@@ -67,7 +67,7 @@ public void start(final ServiceProvider serviceProvider) {
   @Override
   public void stop() {
     statisticsExecutor.shutdownNow();
-    eventsOrderedDeliveryExecutor.shutdownNow();
-    eventsUnorderedDeliveryExecutor.shutdownNow();
+    eventsOrderedDeliveryExecutor.shutdown();
+    eventsUnorderedDeliveryExecutor.shutdown();
```

Comment:
In case of Asynchronous Listeners, this might be required so that a call to `cacheManager.close()` does not cause on going events to stop.